terracotta-kit/src/assemble/init/sysv/terracotta.properties

```diff
@@ -0,0 +1,25 @@
+# Configuration file for the Terracotta SysV init script
+# Intended to be placed into /etc/default/terracotta
+
+# Define some variables
+# Name of app
+APP=terracotta
+
+# Name of the user to run as
+USER=terracotta
+
+# Location of application's bin directory
+TERRACOTTA_HOME=/opt/terracotta
+
+# process name to grep for (in "jps -m -l" output).  Customize if running multiple instances
+JPS_MATCH="com.tc.server.TCServerMain"
+
+# Args to pass to terracotta startup script
+TC_ARGS="-n $APP -f $TERRACOTTA_HOME/config/tc.properties" 
```

Comment:
TC_ARGS="-n $NAME -f $TERRACOTTA_HOME/config/tc-config.xml"

Reply:
what's $NAME ?

Reply:
there are multiple servers specified in the tc-config file.  one of the servers needs to be designated for the invocation.

Reply:
you do that with the -n switch